routing/pathfind.go

```diff
@@ -423,6 +444,79 @@ func newRoute(amtToSend, feeLimit lnwire.MilliSatoshi, sourceVertex Vertex,
 	return route, nil
 }
 
+// NewRouteFromShortHops creates a new Route structure from the minimally
+// required information to perform the payment. It infers fee amounts and
+// populates the node, chan and prev/next hop maps.
+func NewRouteFromShortHops(amtToSend lnwire.MilliSatoshi, timeLock uint32,
```

Comment:
Possible to represent the route returned from path finding as `ShortHop`s also, such that we can reuse this method?